login.html

```diff
@@ -55,27 +46,28 @@
   <body>
     <a href="#container" class="skip-nav">본문 바로가기</a>
     <div id="wrap">
-      <header class="header"></header>
-      <div id="container" class="login-page">
+      <div id="container" class="auth-page">
         <div class="inner03">
-          <a href="" class="login-logo">
+          <a href="/" class="auth-logo">
             <img src="/images/sub/logo_big.svg" alt="판다마켓" />
           </a>
-          <ul class="login-form">
-            <li>
-              <label for="email">이메일</label>
-              <input type="text" id="email" placeholder="이메일을 입력하세요" />
-            </li>
-            <li>
-              <label for="password">비밀번호</label>
-              <input type="password" id="password" placeholder="비밀번호를 입력하세요" />
-              <button type="button" class="toggle-password">
-                <img src="images/common/ic_eye_off.svg" alt="비밀번호 보기" />
-              </button>
-            </li>
-          </ul>
-          <button type="button" class="login-btn">로그인</button>
-          <div class="login-sns">
+          <form action="" onsubmit="login">
```

Comment:
💊 제안
폼 태그의 action 속성은 form 데이터값을 보낼 URI로 아직 배우지 않은 API를 입력해주시는 속성입니다~
지금은 사용하지 않으니 속성을 지우시는 것을 추천드립니다~

```suggestion
          <form onsubmit="login">
```

[mdn form action](https://developer.mozilla.org/ko/docs/Web/HTML/Element/form#action)